### PR TITLE
dont forget to update the docs when adding config changes...

### PR DESCRIPTION
**Pull request by nivong**
more info: https://github.com/PokemonGoF/PokemonGo-Bot/pull/4596

### CODE REVIEW

File: docs/manual_installation.md [hunk 1]
```diff
@@ -78,15 +80,15 @@ pip install -r requirements.txt
 ####
 - finaly start the bot
 ```bash
-./run.sh configs/config.json
```
Comment (mjmadsen) [inline docs/manual_installation.md]: We don't allow passing them in anymore?

Comment (nivong) [inline docs/manual_installation.md]: Why would you if the run.sh does it it self? that is only useful when you have it at a different location and that is not discussed in the manual installation.

Comment (mjmadsen) [inline docs/manual_installation.md]: If you have more than one account.

Comment (nivong) [inline docs/manual_installation.md]: than there should be a second doc for that information not in the manual installation that coffers the manual installation ;)

**Comment by mjmadsen** [comment]
Never mind what I said. We can elaborate on that someplace else.

**Comment by mjmadsen** [comment]
:+1:

**Pull request merged**